common/src/commonMain/sqldelight/com/cuhacking/atlas/db/Database.sq

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE features (
```

Comment:
```suggestion
CREATE TABLE feature (
```
The generated class will be called `Feature` instead of `Features` which will be less confusing when it comes to single objects.
```kotlin
val feature: Features ... // ??
```